Docs/Practitioners/Profile.md

```diff
@@ -0,0 +1,215 @@
+
+## MedicalEducation
+
+Holds information surrounding a person's medical education
+
+| Name | Type   | Description |
+| ---- | ------ | ----------- |
+| school  | [School](#school) | School information |
+| beginDate | DateTime | The date a person started attending  a school |
+| endDate | DateTime | The date a person finished attending a school |
+| degree | [CodedDescription](CodedDescription.md) | An object containing degree information |
+| graduationDate | DateTime | The date the person who attended the school graduated | 
+
+## Name
+
+Holds name information
+
+| Name | Type   | Description |
+| ---- | ------ | ----------- |
+| isLegal  | boolean | Indicates whether the provided name is their legal name |
+| firstName| string | The person's first name |
+| middleName | string | The person's middle name |
+| lastName | string | The person's last name |
+| suffix | string | The suffix the person uses |
+
+## OtherTraining
+
+Holds the other training information of the medical professional
+
+| Name | Type   | Description |
+| ---- | ------ | ----------- |
+| program | [Program](#program) | Program information |
+| specialty | [Specialty](#specialty) | Specialty information |
+| programType | string | The program type |
+| trainingStatus | string | The practitioner's training status |
+| beginDate | DateTime | The date the practitioner started the training |
+| endDate | DateTime | The date the practitioner finished the training |
+
+## Phone
+
+Holds a person's phone number
+
+| Name | Type   | Description |
+| ---- | ------ | ----------- |
+| isPrimary  | boolean | Indicates whether the provided email address is the person's primary |
+| phoneType | string | Indicates the phone type. Ex: Business, Home, Mobile |
+| phoneNumber | string | The person's phone number |
+| extension | string | The extension for the phone number, if necessary |
+
+## Program
+
+Holds a program's relevant data 
+
+| Name | Type   | Description |
+| ---- | ------ | ----------- |
+| hospitalName | string | The hospital's name |
+| affiliatedInstitution | string | The institution the program is affiliated with, if any |
+| city | string | The city where the hospital is located |
+| stateOrProvince | [Region](#region) | The region where the hospital is located |
+
+## Region
+
+Holds fields that describe a region
+
+| Name | Type   | Description |
+| ---- | ------ | ----------- |
+| code  | string | The state or province code |
+| description | string | The name of the state or province |
+| countryCode | string | The country code |
+| countryDescription | string | The name of the country |
+
+## School
+
+Holds information regarding a specified medical school
+
+| Name | Type   | Description |
+| ---- | ------ | ----------- |
+| name  | string | The name of the school |
+| cibisCode | string | The code that Cibis uses to identify the school |
+| schoolType | [CodedDescription](CodedDescription.md) | Degree information |
```

Comment:
Not related to degree. This is the practitioner type for that school (e.g. MD, DO).